plugins/modules/system/xfconf.py

```diff
@@ -95,20 +95,23 @@
     type: str
     sample: "/Xft/DPI"
   value_type:
-    description: The type of the value that was changed
+    description: The type of the value that was changed (None for "get" and "reset" state)
```

Comment:
In jinja2, it is `none`, not `None`.
```suggestion
    description: The type of the value that was changed (C(none) for C(get) and C(reset) state)
```